django/db/models/fields/related.py

```diff
@@ -1885,6 +1891,13 @@ def __init__(self, to, to_field=None, related_name=None, related_query_name=None
             # be correct until contribute_to_class is called. Refs #12190.
             to_field = to_field or (to._meta.pk and to._meta.pk.name)
 
+        if on_delete is None:
```

Comment:
Oh, I see -- because `on_delete` is passed to the `rel` instantiation below. Too bad.